rust-keybroker/keybroker-server/src/main.rs

```diff
@@ -198,9 +202,13 @@ struct Args {
     endpoint: Option<String>,
 
     /// The URL where the verifier can be reached
-    #[arg(long, default_value = "http://veraison.test.linaro.org:8080")]
+    #[arg(long, default_value = "https://veraison.test.linaro.org:8443")]
     verifier: String,
 
+    /// The verifier root certificate to use for a TLS connection
```

Comment:
It should only be supplied if we need to use a root not in the system settings.

```suggestion
    /// The verifier's custom root certificate to use for the TLS connection
```

Reply:
Addressed in the next iteration path (with some an emphasis on the fact this is optional).